lang/pt.js

```diff
@@ -7,25 +7,25 @@ export default {
 	"components.calendar.selected": "Selecionado.",
 	"components.calendar.show": "Mostrar {month}",
 	"components.dialog.close": "Fechar esta caixa de diálogo",
-	"components.form-element.defaultError": "{label} é inválido.",
+	"components.form-element.defaultError": "Valor inválido para {label}.",
 	"components.form-element.defaultFieldLabel": "Campo",
-	"components.form-element.input.email.typeMismatch": "",
-	"components.form-element.input.number.rangeOverflow": "",
-	"components.form-element.input.number.rangeUnderflow": "",
-	"components.form-element.input.text.tooShort": "",
-	"components.form-element.input.url.typeMismatch": "",
-	"components.form-element.valueMissing": "{label} é obrigatório.",
-	"components.form-error-summary.errorSummary": "",
+	"components.form-element.input.email.typeMismatch": "E-mail inválido",
+	"components.form-element.input.number.rangeOverflow": "O número precisa ser menor que {max}",
+	"components.form-element.input.number.rangeUnderflow": "O número precisa ser maior que {min}",
+	"components.form-element.input.text.tooShort": "{label} precisa ter, pelo menos, {minlength} caracteres",
+	"components.form-element.input.url.typeMismatch": "URL inválido",
+	"components.form-element.valueMissing": "O campo {label} é obrigatório.",
```

Comment:
Old: `{label} is required.`
New: `The {label} field is mandatory.`

Old was correct.